src/client/java/teammates/client/scripts/AdminEmailListGenerator.java

```diff
@@ -28,7 +28,7 @@
 
 import com.google.appengine.api.datastore.Key;
 import com.google.appengine.api.datastore.KeyFactory;
-import com.google.appengine.repackaged.org.joda.time.DateTime;
+import com.google.joda.time.DateTime;
```

Comment:
This is a wrong import, as evident from the build log. You should find out what the correct import is.